src/assets/scripts/client/scope/ScopeModel.js

```diff
@@ -205,6 +300,21 @@ export default class ScopeModel {
         return this[functionName](radarTargetModel, ...functionArguments);
     }
 
+    /**
+     * Set the length of the PTL lines for all aircraft
+     *
+     * @for ScopeModel
+     * @method setPtlLength
+     * @param {number} length - length of PTL line, in minutes
+     */
+    setPtlLength(length) {
```

Comment:
I'd almost prefer this param to be called `value` or `nextValue`.

Reply:
Will change to `minutes`

Reply:
done